src/apiCaller.js

```diff
@@ -1,13 +1,26 @@
 import PageData from './components/pageData'
+import path from 'path'
 
 const apiCaller = (opts) => {
   return (files, metalsmith, done) => {
+    const postQuery = (opts.url && opts.url.postBuild) || false
```

Comment:
Perhaps you can update the nodeversion in nvmrc to 14 so you can make use of optional chaining.
You might have to test if there isn't anything that would break when moving to 14.

Reply:
So far -updating the node version is running into issues- even with updates to the `.nvm` and `config.yml` files to reflect the higher node version to support optional chaining -maybe can see if this update can be parked for the moment.

Reply:
Following the node version change to 14, and optional chaining added - I've run tests of the updated branch - using a range of data source types: `hxseo`, `prismic` and `api` - and the pages are building correctly -so testing looks good.